src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
The stylesheet link references a '.scss' file. SCSS files need to be compiled into CSS before they can be used in an HTML document. Ensure that 'styles/index.scss' is compiled to a CSS file and link to that CSS file instead.